tests/taskcluster/bin/graph

```diff
@@ -76,7 +76,7 @@ var currentRunId = process.env.RUN_ID;
 //////////////////////
 // Boostrap the graph
 //////////////////////
-taskIdMap['bootstrap'] = slugid.v4();
+taskIdMap['bootstrap'] = slugid.nice();
```

Comment:
If we can get rid of the dependency for slugid for gaia and instead have that be provided by taskcluster client that would certainly be nice but this also works as is so I will leave it up to the implementer.